### PR TITLE
Update agent.yml with relative imports and cors fix

### PR DESCRIPTION
**Pull request by jaycoolslm**
ref issue: https://github.com/decentralized-identity/veramo-plugin/issues/28

### CODE REVIEW

**Review by nickreynolds — COMMENTED** [review]
Looks great, thanks for doing this! And sorry we hadn't updated this earlier. I'm not sure when this broke as it was working after the Veramo v5 update.

Will wait to approve & merge until @mirceanis takes a look

**Comment by nickreynolds** [comment]
also could you update the PR comment to include "Fixes #28" ?

File: agent.yml [hunk 1]
```diff
@@ -221,7 +221,7 @@ keyManager:
                     - $ref: /constants/secretKey
         # __TEMPLATE__: you can install your own Key Management System implementation too
         my:
-          $require: ./build#MyKeyManagementSystem
+          $require: ./build/key-manager/my-key-management-system.js#MyKeyManagementSystem
```
[the remark this answers is not in the document]
Comment (jaycoolslm) [reply agent.yml:224]: Sure - will test it first to validate that works.

Makes me think that ./build should have worked though... probs just some stupid ESM thingy though

**Comment by jaycoolslm** [comment]
> also could you update the PR comment to include "Fixes #28" ?
Done

Will address the other recommendations tomorrow - getting late over here in UK

**Review by mirceanis — APPROVED** [review]
looks good

**Comment by nickreynolds** [comment]
looks good, going to merge. tests are failing due to goerli deprecation, I can fix that in a separate PR though (tested with sepolia on my end and they pass)

**Comment by jaycoolslm** [comment]
@nickreynolds cheers for checking - was confused as to while failing! But yeah, I remember the Goerli deps now... 

PS. I am getting this error on window OS: `Error creating C:\Users\JC211BT\code\meeco\veramo-plugin\build\index.js['MyKeyManagementSystem']: Only URLs with a scheme in: file, data, and node are supported by the default ESM loader. On Windows, absolute paths must be valid file:// URLs. Received protocol 'c:'. While creating object from pointer: /keyManager. While creating object from pointer: /agent. While creating object from pointer: /server
error Command failed with exit code 1.`

**Pull request merged**